### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#181] [Esercizi - Bertini]
![Screenshot 2023-11-13 194634](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/c0e50c9b-c5ae-4241-a790-f9feee400327)

**Comment by Elia-Belli** [comment]
Siano $X_i\sim Geom(p),i=1,..,6$ le v.a che indicano quanti lanci abbiamo effettuato prima di ottenere una faccia diversa dalle precedenti. Ricordo che per una v.a. geometrica $\mathbb{E}(X)=1/p$

Al primo lancio $\mathbb{E}(X_1)=1$ perchè qualunque faccia esca sarà nuova
Dopo il 1° lancio $\mathbb{P}(faccia\ diversa)=5/6$, allora $\mathbb{E}(X_2)=1/(5/6)=6/5$
Dopo aver trovato 2 facce diverse $\mathbb{P}(faccia\ diversa)=4/6$, allora $\mathbb{E}(X_3)=6/4$ e così via...
In generale $\mathbb{E}(X_i)=6/(7-i)$

La v.a. che indica il numero di lanci per ottenere tutte le facce è equivalente alla somma degli $X_i$ :
Per linearità : $\mathbb{E}(X_1+...+X_6)=\mathbb{E}(X_1)+...+\mathbb{E}(X_6)=1+6/5+6/4+6/3+6/2+6/1=14.7$

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/a3eea502-80b3-4661-9932-c48cd2eb28a3)

**Comment by daromacarlo** [comment]
![Immagine 2023-12-11 113406](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/87928508-4137-4d71-903e-d050797506a0)

**Comment by neoSnakex34** [comment]
ziopera "faccie"
